### PR TITLE
K8 spxc 370 test

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-370](https://jira.percona.com/browse/K8SPXC-370)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -117,6 +121,7 @@ pipeline {
         VERSION = "${env.GIT_BRANCH}-${env.GIT_SHORT_COMMIT}"
         CLUSTER_NAME = sh(script: "echo jenkins-pxc-${GIT_SHORT_COMMIT} | tr '[:upper:]' '[:lower:]'", , returnStdout: true).trim()
         AUTHOR_NAME  = sh(script: "echo ${CHANGE_AUTHOR_EMAIL} | awk -F'@' '{print \$1}'", , returnStdout: true).trim()
+        IMAGE_BACKUP = 'perconalab/percona-xtradb-cluster-operator:master-pxc8.0-backup-test-list'
```
Comment (delgod) [inline Jenkinsfile:124]: Should we remove it?

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-562-badd0401

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-562-465465da

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |

**Pull request closed without merging**